### ISSUE TITLE
it doesn't run at all

### ISSUE THREAD

**Issue by vikram-rawat** [ficonsulting/RInno#98]
Today I updated Rinno from github and created an exe file which installed just fine but when I click on icon it opens a CMD promt and then just fade away. it doesn't open any app. in fact it doesn't do anything at all.

**Comment by vikram-rawat** [comment]
I am installing old version just to finish my work but please look into it. I think something happened with glue package etc.

![image](https://user-images.githubusercontent.com/22342596/45365276-cc11c180-b5f9-11e8-9a57-c8e9b52d21f5.png)

**Comment by r-ballard** [comment]
Hi @vikram-rawat,

The issue appears to be that the old version of RInno (0.1.2) that you installed contains code that was deprecated in an update to tidyverse/glue. In order for the now deprecated glue::collapse function call to work properly you may need to install an older version of the dependency, the 'glue' library.

**Comment by vikram-rawat** [comment]
Thanks for the reply I may do that. But what about the current version of Rinno not working. Is that a known issue or under the hood you are changing something that will fix it within a few days.

**Comment by Dripdrop12** [comment]
@vikram-rawat we did a ton of work recently, so it should be working for you now. Let us know if you are still having an issue

**Comment by vikram-rawat** [comment]
Thanks for updating it. But it asked me to install remotes and pkgbuild and then pkgbuild asked me to install desc which asked me to download from source.

and when I finally did all that it told me 

```
* installing *source* package 'RInno' ...
** R
** inst
** tests
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  namespace 'utils' 3.5.0 is already loaded, but >= 3.5.1 is required
ERROR: lazy loading failed for package 'RInno'
* removing 'C:/Users/201101881/Documents/R/win-library/3.5/RInno'
In R CMD INSTALL
Installation failed: Command failed (1)
```

So I will try again on Monday. but if you could somehow improove the dependency of your own package that would be great help.

But thanks for this anyway.

**Comment by Dripdrop12** [comment]
Thanks for catching this - you can fix this error by updating to R 3.5.1 or installing the newest version of the master branch.
